src/deluge/gui/menu_item/value.h

```diff
@@ -31,12 +32,28 @@ class Value : public MenuItem {
 	void readValueAgain() override;
 	bool selectEncoderActionEditsInstrument() final { return true; }
 
+	void set_value(T value) { value_ = value; }
```

Comment:
for consistency, these should be `camelCase`.

Reply:
Fixed.